pkgs/applications/networking/freefilesync/default.nix

```diff
@@ -96,7 +96,7 @@ gcc12Stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Open Source File Synchronization & Backup Software";
     homepage = "https://freefilesync.org";
-    license = licenses.gpl3Only;
+    license = [ licenses.gpl3Only licenses.openssl licenses.curl licenses.libssh2 ];
```

Comment:
```suggestion
    license = with licenses; [ gpl3Only openssl curl libssh2 ];
```

Reply:
I'm afraid it won't work, since the license names collide with the package names.